core-feature-pack/common/src/main/resources/modules/system/layers/base/org/bouncycastle/bcpkix/main/module.xml

```diff
@@ -32,5 +32,7 @@
     <dependencies>
         <module name="org.bouncycastle.bcprov" services="import"/>
         <module name="org.bouncycastle.bcutil"/>
+        <module name="java.naming"/>
+        <module name="java.sql"/>
```

Comment:
shouldn't this be `java.logging` instead of `java.sql`?

Reply:
Sorry for the mistake. Thank you @soul2zimate :)